src/installer/iohandler.py

```diff
@@ -0,0 +1,100 @@
+"""Handling wheel installation I/O-related functionality."""
+
+import hashlib
+import os
+from io import RawIOBase
+
+from installer._compat.typing import TYPE_CHECKING
+
+from .records import Hash, Record
+
+if TYPE_CHECKING:
+    from ._compat.typing import FSPath
+    from typing import Dict, Tuple, Optional, Callable
+
+_BLOCK_SIZE = 4096
+_SCHEME_KEYS = ["platlib", "purelib"]
```

Comment:
This is not being used anywhere. Leftover?

Reply:
Yea, looks like. We'd probably want to use `SCHEME_KEYS` from `installer.utils`.

Reply:
Yup, good catch!